pages.ml/linux/nmcli.md

```diff
@@ -0,0 +1,19 @@
+# nmcli
+
+> നെറ്റ്‌വർക്ക് മാനേജർ നിയന്ത്രിക്കാൻ വേണ്ടിയുള്ള കമാൻഡ് ലൈൻ ഉപകരണം.
+
+- nmcli പതിപ്പ് ഏതാണെന്ന് അറിയാൻ :
+
+`nmcli --version`
+
+- പൊതുവെയുള്ള സഹായ വിവരം കാണാൻ :
+
+`nmcli --help`
+
+- ഒരു പ്രതേക നിർദേശത്തിന്റെ സഹായ വിവരം കാണാൻ :
```

Comment:
I don't know the language, but I have to check this translation because the term `നിർദേശം` is not present in this example description, and looks `നിർദേശം` is the translation for `command`:
![Screenshot_20201013_080348](https://user-images.githubusercontent.com/1536740/95852759-a56b6b00-0d2a-11eb-9e57-56d587450d63.png)

Reply:
Hi, 
I was using this [file](https://github.com/tldr-pages/tldr/blob/master/pages/linux/nmcli.md) for translation. 
The third example given in that file is ` Call help on a command ` with the command as `nmcli {{command}} --help` . So I translated it as such.
I didn't find the `nmcli {{suggestion}} --help` example in the pages/linux/nmcli.md file.

Reply:
No, gTranslation (for some reason that I don't know) translation `command` in your language to `instruction` or `suggestion`.
But you're translating `Call help on a command` and I can't see the word `നിർദേശം`. So I just want to check if the example description is good translation or have a typo

Reply:
It's not a typo. `നിർദേശത്തിന്റെ` means "command's" .